### DISCUSSION TITLE
Gold bar coin pile tweaking

### DISCUSSION THREAD

**Discussion by Butters1984** [Digitalroot-Valheim/Digitalroot.Valheim.GoldBars#9] [Ideas]
is it possible to be able to edit coinpile to 999 goldcoins to create 1 gold bar = 999?

**Comment by Digitalroot** [comment]
I could do that.

**Reply by Butters1984** [reply]
@Digitalroot I would really appreciate and love you for that.

**Reply by Butters1984** [reply]
and retain weight of 20 please?
